mysite/polls/views.py

```diff
@@ -0,0 +1,43 @@
+from django.shortcuts import render
+import search
+import json
+from django.http import JsonResponse
+from django.http import HttpResponse
+import re
+# Create your views here.
+
+
+def make_link_for_html(movie_id):
+    link = '<a href = "http://localhost:8000/polls/info/?q=' \
```

Comment:
you should prefer wrapping long lines within parens than doing this. Read https://www.python.org/dev/peps/pep-0008/#maximum-line-length